test/knex.test.js

```diff
@@ -0,0 +1,75 @@
+const seneca = require('seneca')
+// const Shared = require('seneca-store-test')
+const Lab = require('@hapi/lab')
+const lab = (exports.lab = Lab.script())
+const { describe, it, before } = lab
+const { expect } = require('@hapi/code')
+
+const KnexStore = require('../src/knex-store')
+const DbConfig = require('./config/database/config')
+
+describe('knex-store tests', function () {
+  const test = seneca()
+    .test()
+    .use('entity')
+    .use(KnexStore, { DbConfig })
+    .use('promisify')
+
+  console.log(test)
+
+  before(() => {
+    return new Promise((done) => {
+      seneca().ready(done)
+    })
+  })
+
+  it('save', async () => {
+    const s0 = await seneca().entity().begin()
+    console.log(s0)
+    const foo1 = await s0.entity('foo').data$({ x: 1 }).save$()
```

Comment:
In this test, in addition to the existing code, - please make an explicit call to the database using `knex` directly, and verify that the `foo` entity has been saved with the correct data, e.g.:
```javascript
const foo_record = await knex('foo').first()
expect(foo_record).to.be.an.object()
expect(foo_record.x).to.equal(1)
```

Reply:
Ok